azure/inputsources/shared/masterdata.ftl

```diff
@@ -853,7 +853,8 @@
                                 "oai": {
                                     "Engine": "oai"
                                 }
-                            }
+                            },
+                            "azure:AdministratorGroups": []
```

Comment:
Not required, the default value is set on the attribute anyway

Reply:
The default value causes an error to be raised ("no administrator group"). We're trying to find a better default value so the default "just works", so this is part default value, part placeholder for a better default value. The default on the attribute is to avoid any crashes (as distinct from deliberate error reporting)

Reply:
Where are you seeing an error? It might be better to check for content in the code which uses it and if there isn't content skip the step. This would then allow the validation failure to get out which is probably where the issue is at the moment.